foo_spider_monkey_panel/js_objects/gdi_bitmap.cpp

```diff
@@ -73,6 +74,7 @@ constexpr auto jsFunctions = smp::to_array<JSFunctionSpec>(
         JS_FN( "GetColourScheme", GetColourScheme, 1, kDefaultPropsFlags ),
         JS_FN( "GetColourSchemeJSON", GetColourSchemeJSON, 1, kDefaultPropsFlags ),
         JS_FN( "GetGraphics", GetGraphics, 0, kDefaultPropsFlags ),
+        JS_FN( "InvertColours", InvertColours, 1, kDefaultPropsFlags ),
```

Comment:
`InvertColours` does not accept any arguments, so this should be zero:
```suggestion
        JS_FN( "InvertColours", InvertColours, 0, kDefaultPropsFlags ),
```